website/package.json

```diff
@@ -43,7 +43,7 @@
     "@docusaurus/module-type-aliases": "^3.0.0",
     "@tsconfig/docusaurus": "^2.0.2",
     "@typescript-eslint/eslint-plugin": "^6.13.1",
-    "@typescript-eslint/parser": "^6.20.0",
+    "@typescript-eslint/parser": "^7.1.0",
```

Comment:
In order to fix this version bump, both `eslint-plugin` and `parser` must be upgraded to `7.1.0`.

I don't have js tools to do this now.